src/serve/genericWorker.ts

```diff
@@ -0,0 +1,26 @@
+import { sbp } from '~/deps.ts'
+import { parentPort } from 'node:worker_threads'
+import { initDB } from './database.ts'
+import './logger.ts'
+
+export const readyQueueName = 'parentPort'
+
+parentPort!.on('message', ([port, ...msg]: [MessagePort, ...unknown[]]) => {
+  sbp('okTurtles.eventQueue/queueEvent', readyQueueName, () => {
+    (async () => {
+      try {
+        // The `any` cast is not ideal, but sbp's dynamic nature makes
+        // this hard to type statically without more complex infrastructure.
+        port?.postMessage([true, await sbp(...(msg as [string, ...unknown[]]))])
+      } catch (e) {
+        const err = e as Error
+        port?.postMessage([false, { message: err.message, stack: err.stack }])
```

Comment:
Why is this being changed?

Reply:
Hm, I see now. So, you're saying that `sbp(...args)` doesn't work, so you need type casting, right? That's fine.